new-website/utils/tutorials/build_pdf_book.py

```diff
@@ -0,0 +1,64 @@
+"""
+This script is used to build the pdf book from DeepChem Tutorials.
+
+Requirements:
+    - pdfunite
```

Comment:
Add this to our requirements.txt

Reply:
pdfunite is a linux application which i am using to merge pdfs.
I added pdfkit to requirements.txt.